components/frontend/src/widgets/Button.js

```diff
@@ -115,7 +115,9 @@ export function AddDropdownButton({ itemSubtypes, itemType, onClick, allItemSubt
         items = items.filter((item) => !usedItemSubtypeKeys.includes(item.key))
     }
     const options = items.filter((itemSubtype) => itemSubtype.text.toLowerCase().includes(query.toLowerCase()))
-    options.sort((a, b) => a.text.localeCompare(b.text))
+    if (sort !== false) {
```

Comment:
Why the double negate? Prop type says it's bool so there shouldn't be more options than true or false right?

Reply:
Prop types are optional by default, and evaluating the sort parameter this way makes its default value true.